src/collector.js

```diff
@@ -0,0 +1,53 @@
+import { default as Message } from 'message';
+
+// "I have a display case ready and waiting for our newest acquistions!"
+// --Taneleer Tivan
+
+
+export default class Collector {
+
+  constructor() {
+    this.errors = [];
+    this.notices = [];
+    this.warnings = [];
+  }
+
+  getMessageList(type) {
```

Comment:
Oh, I guess it allows you to use the type constants (`collection.getMessageList('error')` instead of `collection.errors`).

At the very least I think `getMessages` would be nicer, but I find a plural (`getMessages`) method name enough to imply I get back an array. If that's not explicit enough though I understand.